person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def intialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    if @age >= 18
+      true
+    elsif @age > 18
```

Comment:
- [ ] Kindly change the condition of else if to be `@age < 18` in order to make the logic right.
- [ ] [OPTIONAL] You can simply implement the method by returning `@age >= 18` so it will be like:
```
def of_age?
    @age >= 18
end
```